modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -480,6 +495,101 @@ inline IE::Blob::Ptr extractBlob(IECallContext& ctx, std::size_t i) {
 }
 } // anonymous namespace
 
+class cv::gimpl::ie::RequestPool {
+public:
+    using CompiledPtr = std::shared_ptr<cv::gimpl::ie::IECompiled>;
+    using Run      = std::function<void(cv::gimpl::ie::IECompiled&)>;
+    using Callback = std::function<void(cv::gimpl::ie::IECompiled&)>;
```

Comment:
I'd put `F` like `RunF` to make it clear it is a function

Reply:
Fixed